src/borg/testsuite/archiver.py

```diff
@@ -2561,6 +2571,7 @@ def test_recreate_dry_run(self):
         archives_after = self.cmd('list', self.repository_location + '::test')
         assert archives_after == archives_before
 
+
```

Comment:
remove accidental change, please.

Reply:
this is why the travis flake8 test is failing currently.

Reply:
Sure, it is done.
My bad, I forgot to re-run this test in the end.